examples/deepzoom/deepzoom_tile.py

```diff
@@ -193,15 +209,14 @@ def _url_for(self, associated):
             base = VIEWER_SLIDE_NAME
         else:
             base = self._slugify(associated)
-        return '%s.dzi' % base
+        return "%s.dzi" % base
 
     def _write_html(self):
         import jinja2
-        env = jinja2.Environment(loader=jinja2.PackageLoader(__name__),
-                    autoescape=True)
-        template = env.get_template('slide-multipane.html')
-        associated_urls = {n: self._url_for(n)
-                    for n in self._slide.associated_images}
+
+        env = jinja2.Environment(loader=jinja2.PackageLoader(__name__), autoescape=True)
```

Comment:
It looks as though we're giving up on 80-column line lengths.  I think that may be a reasonable tradeoff; there are various places in the codebase with contorted formatting to fit within that limit.  But we should be clear that we want to do that before proceeding.